apps/src/templates/rubrics/LearningGoal.jsx

```diff
@@ -279,11 +279,7 @@ const AiToken = () => {
     <div>
       {' '}
       <BodyFourText className={classnames(style.aiToken, style.aiTokenText)}>
-        <ExtraStrongText>
-          {i18n.artificialIntelligenceAbbreviation()}
-        </ExtraStrongText>
-
-        <FontAwesome icon="check" title={i18n.aiAssessmentEnabled()} />
```

Comment:
Should these strings be removed as well?

Reply:
They are not used anywhere else... I will remove them!